chainerx_cc/chainerx/kernels/linalg.h

```diff
@@ -39,4 +40,11 @@ class SvdKernel : public Kernel {
     virtual void Call(const Array& a, const Array& u, const Array& s, const Array& vt, bool full_matrices) = 0;
 };
 
+class SyevdKernel : public Kernel {
+public:
+    static const char* name() { return "Syevd"; }
+
+    virtual void Call(const Array& a, const Array& w, const Array& v, const std::string& UPLO, bool compute_eigen_vector) = 0;
```

Comment:
How about plural "compute_eigen_vectors"? "compute_v" is an option too.

Reply:
I changed to be `compute_v`, so it similar to SVD's `compute_uv`.